src/main/java/org/jitsi/jigasi/AbstractGatewaySession.java

```diff
@@ -261,9 +261,9 @@ void notifyChatRoomMemberJoined(ChatRoomMember member)
      *
      * @param member the member who left the JVB conference
      */
-    // FIXME: 17/07/17 JvbConference does not yet call this method
     void notifyChatRoomMemberLeft(ChatRoomMember member)
     {
+        participantsCount--;
```

Comment:
Wait, didn't you say that it shouldn't decrement, because it counts the number of participants that joined the call, not the number of current participants?

Reply:
I cleared it up with damyan and it should indeed NOT decrement

Reply:
Its me that brought the confusion thinking of some stats and without reading the javadoc where it is written:
```
     * Returns the cumulative number of participants that were active during
     * this session including the focus.
```